app/views/providers/check_provider_answers/index.html.erb

```diff
@@ -0,0 +1,43 @@
+<% content_for :navigation do %>
+  <%= link_to (t 'generic.back'), edit_providers_legal_aid_application_applicant_path, class: 'govuk-back-link' %>
```

Comment:
**[nit pick/not a blocker]** Was this supposed to be `link_to t('generic.back'),` ? The brackets surrounding the translation seem unnecessary.

Reply:
I've seen this is actually across the whole template.

Reply:
👍